packages/backend-modules/mail/templates/future_campaign_sender_reward.html

```diff
@@ -81,68 +88,90 @@
                       {{receiver}}!
                     </p>
                     <p>
-                      Damit haben Sie noch {{count_open_slots}} Zugänge zu
-                      vergeben.
+                      Zwei ist nicht nur die kleinste und einzig gerade
+                      Primzahl – sondern auch die Zahl der Arme, der Arche, der
+                      Liebe.
+                    </p>
+                    <p>
+                      Wobei, manchmal ist zwei auch zu wenig. Im Fall der Arme
+                      etwa bei den Tintenfischen, im Fall der Liebe im
+                      französischen Film, und im Fall der Arche sang Tom Combo
+                      einmal zu Recht: «Bakterien waren auf der Arche Noah – und
+                      es waren mehr als zwei.»
+                    </p>
+                    <p>
+                      Was im Fall der Republik heisst: Sie haben noch
+                      {{count_open_slots}} Zugänge zu vergeben.
                     </p>
                     <p>
                       Vielleicht wäre das etwas für Ihren Coiffeur, Ihre
-                      Skilehrerin oder Ihren Hausarzt?
+                      Skilehrerin, Ihre Yoga- oder Pokerrunde, Ihren Hausarzt?
                     </p>
                     <p>
-                      <a href="{{link_sender_page}}">Finden Sie es heraus</a>!
```

Comment:
Link should not be in a separate paragraph. Should stay the same as before. (in the old version it is the very same)